docs/generated/templates.md

```diff
@@ -613,6 +613,39 @@ KubeLinter supports the following templates:
 ]
 ```
 
+## Untrusted images
+
+**Key**: `untrusted-images`
+
+**Description**: Flag applications running containers with untrusted container images
+
+**Supported Objects**: DeploymentLike
+
+**Parameters**:
+
+```json
+[
+  {
+    "name": "allowedList",
+    "type": "array",
+    "description": "list of allowed container image registries",
+    "required": false,
+    "regexAllowed": true,
+    "negationAllowed": true,
+    "arrayElemType": "string"
+  },
+  {
+    "name": "blockedList",
```

Comment:
Nit: it seems more in line with industry practice to call this `allowlist` and `blocklist`. (Since it's one word, we should lower-case the `L` in `list`)

Reply:
noted. Thank you!